frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/active-health-checks.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+import { MINUTE } from '../consts';
+
+describe('Test Popover behaviour for different active health check cases.', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    cy.install();
+    cy.visit('/ocs-dashboards/block-file');
```

Comment:
This should be selected via data-test ids. Using url links is not long term solution. Can you take a look ?

Reply:
Also create a separate method to visit these links